R/reporter.R

```diff
@@ -13,10 +13,12 @@ Reporter <- R6::R6Class("Reporter",
     start_reporter = function() {},
     start_context =  function(context) {},
     start_test =     function(context, test) {},
+    start_file =     function(filename) {},
     add_result =     function(context, test, result) {},
     end_test =       function(context, test) {},
     end_context =    function(context) {},
     end_reporter =   function() {},
+    end_file =       function() {},
```

Comment:
I added `end_file` even though I don't actually use it because we may need it in the future if we are removing context in favor of filenames.